tools/js2c.cc

```diff
@@ -532,8 +539,7 @@ Fragment GetDefinitionImpl(const std::vector<char>& code,
   // Avoid using snprintf on large chunks of data because it's much slower.
   // It's fine to use it on small amount of data though.
   if constexpr (is_two_byte) {
-    std::vector<uint16_t> utf16_codepoints;
-    utf16_codepoints.resize(count);
```

Comment:
Can this be the reason of the Windows issue? I wonder if the implementation differs in Windows? @lemire

Reply:
@anonrig You replaced a default constructor + a resize with a proper constructor. I can't imagine it would matter.